posts/no-uncle-of-mine.md

```diff
@@ -0,0 +1,99 @@
+him :
+
+> You never punish bad ideas.
+> Instead, you counter bad ideas with better ideas.
+
+And as such, not only was the firing a wrong decision, but it would in turn make
+people unconfortable sharing their ideas, which would endanger a company as
+Google that relies on innovation.
+
+While I agree that you should try to discuss things freely, and allow people and
+their ideas to evolve though debate, there is a limit to how far this can go on
+for. In this particular context, it seems like Google did provide
+opportunities both for discussion and learning, as is mentionned in the
+manifesto itself. If even after this, someone continues pushing on a subject
+with a rhetoric that hurts the company and its employees, I don't see any other
+path other than the dismissal of the offender.
+
+Furthermore, it is important indeed that people feel like they can share their
+ideas freely. So this raises the question, what should a company do when one of
+its employees says half of his coworkers are less suited for the job than him?
+History suggests that letting this kind of thinking foster inside a company
+could make the people targetted by that memo less confident in themselves and
+hence less prone to share their thoughts.
+
+Finally, while a company like Google does rely on innovation to continue its
+business, there is nothing new in that manifesto. The points Damore made were
+old ones, and you would be excused for hoping these would be considered defeated
+in the 21st century.
+
+## The dangers of missing the point
+
+Some things are not okay to say, especially in an industry such as ours where
+white men are over represented. Thus the need for our community, in particular
+its more privileged members, to stand up and denounce points of view such as the
+one expressed by Damore. Failure in doing so will continue to make this community
+unwelcoming for under representended folks.
+
+This is why Stan's stance was probably the most toxic thing a prominent white
+men in the community could have said. As such Stan should stop complaining
+about the quality of the feedback he received, and instead start asking himself
+some questions as to why people sent him mostly aggressive or condescending
+tweets.
+
+Part of the issue also is that Stan has a pretty poor track record when it comes
+to talking about women in tech, and one of his following posts sheds some light
+into why that is. While he has recognized and publicly apologized for some
+misplaced humour in the past, he seems very unapologetic about this current
+issue.
+
+## Greater than code
+
+When trying to wrap my head around this, I realized something about Stan. One
+of the reasons he is so important in the software industry is because of his
+thoughts on the way we work. One thing that nows strikes me is how much emphasis
+he puts on professional skills. While this emphasis has many values, bore many
+fruits, and has an historical reason to come into existence, I start feeling
```

Comment:
a historical

Reply:
I prefer the "an" variant of this. Both are acceptance AFAICR

Reply:
+1 for "a historical". Unless we're dropping the H and pronouncing it "anistorical".